app/js/components/setsOrderEntry/OrderSetForm.jsx

```diff
@@ -31,30 +35,23 @@ class OrderSetForm extends PureComponent {
           <div>
             <h1>Order from Sets</h1>
           </div>
-          <span style={{ display: displaySets }}>
+          <span style={{ display: displaySets ? 'block' : 'none' }}>
             <SelectBox
               items={items}
               selectedItem={selectedItem}
               onChange={this.changeSelectedOrderSet}
             />
           </span>
-          <span style={{ display: displayForm }}>
+          <span style={{ display: displayForm ? 'block' : 'none' }}>
             <div className="set-header">
               <h1>DIABETES INITIAL WORKUP</h1>
             </div>
             <table>
               <tbody>
                 <tr className="set-details">
-                  <th className="set-details-header">Details</th>
-                  <th className="set-actions-header">Actions</th>
+                  <th colSpan="2" className="set-details-header">Details</th>
                 </tr>
                 <Accordion
-                  title={
```

Comment:
Is this deletion part of this ticket?

Reply:
Yes

Reply:
@Daramola98 Please delete it from the imports as well then. If its no longer needed.